src/soar_sdk/app.py

```diff
@@ -16,19 +16,57 @@
 from soar_sdk.params import Params
 from soar_sdk.action_results import ActionOutput
 from soar_sdk.types import Action, action_protocol
+import uuid
+
+
+def is_valid_uuid(value: str) -> bool:
+    """Validates if a string is a valid UUID"""
+    try:
+        return str(uuid.UUID(value)) == value
```

Comment:
Let's make this case-insensitive
```suggestion
        return str(uuid.UUID(value)).lower() == value.lower()
```